conf/modules.config

```
@@ -249,3 +249,23 @@ if (!params.skip_fastp) {
         }
     }
 }
+
+if (params.annotation_tool == 'bakta') {
+    if (params.baktadb_download == true) {
+        process {
+            withName: '.*:.*:BAKTA_DBDOWNLOAD_RUN:BAKTA_BAKTADBDOWNLOAD' {
+                ext.args = params.baktadb_download_args ? params.baktadb_download_args : ''
+            }
+        }
+    }
+    process {
+        withName: '.*:.*:BAKTA_DBDOWNLOAD_RUN:BAKTA_BAKTA' {
+            ext.args    = ''
+            publishDir  = [
+                path: { "${params.outdir}/Bakta" },
```

```suggestion
                path: { "${params.outdir}/Bakta/${meta.id}" },
```
this should make `--output $prefix` and a bunch of other changes in the Bakta module unnecessary. An example can be found [here](https://github.com/nf-core/mag/blob/ba7234959405b7a6df253d6d03158a7cde030c96/conf/modules.config#L44).

---

yep, it is much better than patching a nf-core module. I didn't know that meta.id work in modules.config